rafx-framework/Cargo.toml

```diff
@@ -27,6 +29,8 @@ profiling = "0.1.9"
 serde = { version = "1", features = ["serde_derive"] }
 serde_bytes = "0.11"
 glam = { version = "0.13.1", features = [ "serde" ] }
+parking_lot = "0.11.1"
+slotmap = "1.0.2"
```

Comment:
I want to look into slotmap a bit more later, seems similar to the GenSlab this PR removes (but maybe it does more stuff or implements things in a better way than GenSlab)